src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -55,6 +56,16 @@ public ReadPileup(final Locatable loc, final List<GATKRead> reads, final List<In
     }
 
     /**
+     * Returns the element corresponding to the given read of null there is no such element.
+     * @param read or null if elements with no reads are to be retrieved from this pileup.
```

Comment:
Can there be an element with no read? What if there are multiple such elements? (document)